### PR TITLE
Create Candidates-2023

### PR DESCRIPTION
**Pull request by DavidPHirsch**
adding files and process for governance committee elections

### CODE REVIEW

**Comment by DavidPHirsch** [comment]
@beeme1mr can you also share this in the GC slack channel?

**Comment by DavidPHirsch** [comment]
@open-feature/governance-board

**Review by moredip — APPROVED** [review]

**Review by beeme1mr — APPROVED** [review]

**Review by dabeeeenster — APPROVED** [review]

File: Elections/2023/Candidates.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+The 4 members that are elected from the current governance board will have a one year term and and the 3 elected from the community will have a two year term.
+
+## Current Governance committee members (4 seats open, with 1 year term)
+
+- [Alois Reitbauer](https://github.com/aloisreitbauer), Dynatrace
+- [Ben Rometsch](https://github.com/dabeeeenster), Flagsmith
+- [Justin Abrahms](https://github.com/justinabrahms), eBay
+- [Michael Beemer](https://github.com/beeme1mr), Dynatrace
+- [Pete Hodgson](https://github.com/moredip), Independent
+- [Alex Jones](https://github.com/AlexsJones), Canonical
```
Comment (AlexsJones) [inline Elections/2023/Candidates.md:13]:
```suggestion
- [Alex Jones](https://github.com/AlexsJones), AWS
```
TY!

**Review by AlexsJones — CHANGES_REQUESTED** [review]

**Comment by AloisReitbauer** [comment]
This is not the same model as for OpenTelemetry? Is it?

**Pull request merged**